company-profiles/17hats.md

```diff
@@ -22,4 +22,4 @@ iOS, React, Knockout, Rails, Perl, HTML, Sql, Ruby, JQuery
 
 ## How to apply
 
-Email buford@17hats.com with github and/or CV
+Email buford@17hats.com with github-ID and/or CV
```

Comment:
For such a small change can we make this better, please? GitHub rather than `github` and update it to something like "Email burford@17hats.com your GitHub profile and/or your CV."

Thanks.

Reply:
okk sir I will do it ASAP

Reply:
Sir i have done some changes please see to it